app/src/pages/clinic.tsx

```diff
@@ -19,9 +20,28 @@ const Clinic: NextPage = () => {
   const handleSearch = (e: FormEvent<HTMLFormElement>) => {
     e.preventDefault()
     setSelectedClinic(undefined)
-    const filtered = clinics.filter((clinic) => clinic.zip === search)
 
-    setFilteredClinics(filtered)
+    if (search) {
+      import(
+        `../../public/clinic-output/clinics-zip-code-lookup/${search}.json`
+      )
+        .then(
+          (sortedClinics: { default: { id: number; distance: string }[] }) => {
+            const clinicsWithDetails: typeof clinics = sortedClinics.default
```

Comment:
```suggestion
            const clinicsWithDetails: (typeof clinics[0] | undefined)[] = sortedClinics.default
```
This will fix the type error we were seeing